src/gms/generate_branch_list.py

```diff
@@ -1,36 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
This file was deleted but being used elsewhere.

Reply:
restored. thank you.